index.js

```diff
@@ -1,19 +1,39 @@
+const yaml = require('js-yaml');
+
 module.exports = robot => {
     robot.on('pull_request.opened', receive);
     robot.on('issues.opened', receive);
     async function receive(context) {
-        const title = context.payload.issue.title;
-        const body = context.payload.issue.body;
-        // robot.log('Title: ', title, 'Body: ', body);
-
-        // Add options for if there is a default title?
+        let title, body, number, bodies, badTitle;
+        if (context.payload.pull_request) {
+            ({ title, body, number } = context.payload.pull_request);
+        } else {
+            ({ title, body, number } = context.payload.issue);
+        }
 
-        if (!body) {
-            const options = context.repo({path: '.github/request-info.md'});
+        try {
+            const options = context.repo({path: '.github/config.yml'});
             const response = await context.github.repos.getContent(options);
-            const template = Buffer.from(response.data.content, 'base64').toString();
+            bodies = yaml.load(Buffer.from(response.data.content, 'base64').toString()) || {};
+        } catch (err) {
+            if (err.code !== 404) throw err;
+        }
 
-            context.github.issues.createComment(context.issue({body: template}));
+        if (bodies) {
+            if (bodies.requestInfoDefaultTitles.includes(title.toLowerCase())) badTitle = true;
+        }
+        if (!body || badTitle) {
+            context.github.issues.createComment(context.issue({body: bodies.requestInfoReplyComment}));
+            if (bodies.requestInfoLabelToAdd) {
+                // Add label if there is one listed in the yaml file
+                const repo = context.payload.repository;
+                context.github.issues.addLabels({
+                    owner: repo.owner.login,
+                    repo: repo.name,
+                    number,
```

Comment:
The linter expects property shorthand: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer